cmd/context/create.go

```diff
@@ -380,6 +380,17 @@ func (c ContextCommand) getUserContext(ctx context.Context, ns string) (*types.U
 				return nil, fmt.Errorf(oktetoErrors.ErrCorruptedOktetoContexts, config.GetOktetoContextsStorePath())
 			}
 		}
+
+		kubetoken, err := client.Kubetoken().GetKubeToken(okteto.Context().Name, okteto.Context().Namespace)
```

Comment:
Are we managing the lifecycle of the token appropriately? This is an api call that is potentially expensive on the backend and also blocking if the token cannot be retrieved.

The token should be good for 1d or a length of time. Do we *always* retrieve the token for every command that runs or is the token gone? What are the failure modes for when we possibly could have a good token and use it over getting a new one either b/c it's very new or the api is not working? In these scenarios what does the kubectl lib do or the AKS open source implementation and are we inline with that.

I want to avoid creating a full incident and outage when we only really need to have a partial outage in the case of the api being down for a bit. It's also possible that this is irrelevant b/c we only need this kube token when we're making api calls anyways?